test/e2e/framework/framework.go

```diff
@@ -117,6 +118,7 @@ func NewFromFlags() (*Framework, error) {
 	stressTestLevel := flag.Int("stress", 0, "enable stress test at given level 0-100")
 	perfOutputDir := flag.String("perf-output", "", "write performance statistics to the specified directory")
 	version := flag.String("version", "", "agones controller version to be tested, consists of release version plus a short hash of the latest commit")
+	featureGates := flag.String("featuregates", "", "list of feature gates to be tested in the test run")
```

Comment:
Should this be the same as https://github.com/googleforgames/agones/blob/master/pkg/util/runtime/features.go#L66 ?

We have `featureGateFlag` variable set in there - would make sense to keep it all the same. Then we can reuse [ParseFeaturesFromEnv](https://github.com/googleforgames/agones/blob/master/pkg/util/runtime/features.go#L79) as well.

(Long term, should we move this to pflag?)